frontend/src/stores/search.ts

```diff
@@ -333,9 +333,39 @@ export const useSearchStore = defineStore("search", {
 
       this.addRecentSearch(formattedTerm)
     },
+    isCollectionStateSame(
+      collectionParams: CollectionParams,
+      mediaType: SupportedMediaType
+    ) {
```

Comment:
Is there an easy way to test this new function?

Reply:
I removed this function in favor of @wordpress/is-shallow-equal that we are already using in another component.